tests/e2e/deployment/deployment_test.go

```diff
@@ -55,26 +55,164 @@ var _ = Describe("Application deployment in edge_core Testing", func() {
 			utils.PrintTestcaseNameandStatus()
 		})
 
-		FIt("E2E_APP_DEPLOYMENT_1: Test application deployment in edge_core", func() {
+		It("E2E_APP_DEPLOYMENT_1: Test application deployment in using deploymet controller", func() {
 			var deploymentList v1.DeploymentList
 			var podlist metav1.PodList
 			replica := 1
 			//Generate the random string and assign as a UID
 			UID = "deployment-app-" + utils.GetRandomString(5)
-			IsAppDeployed := utils.HandleDeployment(http.MethodPost, ctx.Cfg.ApiServer+DeploymentHandler, UID, ctx.Cfg.AppImageUrl[1], replica)
+			IsAppDeployed := utils.HandleDeployment(http.MethodPost, ctx.Cfg.ApiServer+DeploymentHandler, UID, ctx.Cfg.AppImageUrl[1], nodeSelector, replica)
 			Expect(IsAppDeployed).Should(BeTrue())
-			//CheckPodRunningState(ctx.Cfg.ApiServer+AppHandler, UID)
 			err := utils.GetDeployments(&deploymentList, ctx.Cfg.ApiServer+DeploymentHandler)
 			Expect(err).To(BeNil())
 			for _, deployment := range deploymentList.Items {
 				if deployment.Name == UID {
-					label := deployment.ObjectMeta.Labels["app"]
+					label := nodeName
 					podlist, err = utils.GetPods(ctx.Cfg.ApiServer+AppHandler, label)
 					Expect(err).To(BeNil())
 					break
 				}
 			}
 			utils.CheckPodRunningState(ctx.Cfg.ApiServer+AppHandler, podlist)
 		})
+		It("E2E_APP_DEPLOYMENT_2: Test application deployment in using deploymet controller", func() {
```

Comment:
It should be described like : 

It("Runs deployments and verifies whether the pods come up correctly")

We are describing what the test does, we can make it more readable.